js/src/common/helpers/avatar.tsx

```diff
@@ -1,26 +1,28 @@
+import * as Mithril from 'mithril';
+import User from '../models/User';
+
 /**
  * The `avatar` helper displays a user's avatar.
  *
- * @param {User} user
- * @param {Object} attrs Attributes to apply to the avatar element
- * @return {Object}
+ * @param attrs Attributes to apply to the avatar element
```

Comment:
I wouldn't want to have one param documented but the other not

Reply:
The other is still documented in the typehinting. Attrs only remains for the description in the JSDoc. I don't know if we want to do that or keep tham all, howeverr 🤷

Reply:
I think we should leave all parameters in JSDoc if there's any with description and get rid of all parameters if none of them have any description. By the way this should resolve PHPStorm's warning "Parameter is not described in JSDoc".